doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -429,7 +429,8 @@ Linear algebra equivalents
      - concatenate columns of ``a`` and ``b``
 
    * - ``[a; b]``
-     - ``np.concatenate((a,b))`` or ``np.vstack((a,b))`` or ``np.r_[a,b]``
+     - ``np.concatenate((a,b))`` or ``np.vstack((a,b))`` or
+       ``np.row_stack((a,b))`` or ``np.r_[a,b]``
```

Comment:
This is correct, but IMO it'd be better not to advertise `row_stack` given that it's just a direct alias of `vstack` (it even uses `vstack`'s docstring!).

Not really important though, if anyone feels strongly it should be included then :+1:.

Reply:
It might be nice to mention `np.stack` since it is (to some degree), the preferred function nowadays probably.

Reply:
(But, my guess is that there are a lot of improvements here, so we could also just go ahead and merge).

Reply:
Ok, good, I was wondering about `row_stack`, because it's referring to itself as `vstack` in the docstring. I can revert that and mention `stack`.

Reply:
Actually, `stack` only does the equivalent thing for 1-D arrays, not 2-D arrays, so I'm not sure about adding it. I removed `row_stack`.